README.md

```diff
@@ -1,68 +1,66 @@
 
-- one for the season start *tthrs/tthre*
-- one for the tuning *tune*
-- for the season end *tthre/saisl*
+```bash
+tools/setup_miniconda.sh -p $SCRATCH -u
+```
 
-## Plotting
+In case you ever need to uninstall miniconda, do the following:
 
-In the /notebook folder is a simple script that allows for plotting 2D-maps using xarray and iconarray.
+```bash
+conda init --reverse --all
+rm -rf $SCRATCH/miniconda
+```
 
-## Start developing
+## Ho to install the package
 
-Once you created or cloned this repository, make sure the installation is running properly. Install the package dependencies with the provided script `setup_env.sh`. This script also handles the installation of ecCodes cosmo definitions (for more on ecCode refer to the dedicated section below), sets the fieldextra path and checks the cartopy installation (currently deactivated).
+Once you created or cloned this repository, make sure the installation is running properly. Install the package dependencies with the provided script `setup_env.sh`. This script also handles the installation of ecCodes cosmo definitions (for more on ecCode refer to the dedicated section below) and sets the fieldextra path.
 Check available options with
 
 ```bash
 tools/setup_env.sh -h
 ```
-
-We distinguish development installations which are editable and have additional dependencies on formatters and linters from productive installations which are non-editable
-and have no additional dependencies. Moreover we distinguish pinned installations based on exported (reproducible) environments and free installations where the installation
-is based on top-level dependencies listed in `requirements/requirements.yml`. If you start developing, you might want to do an unpinned installation and export the environment:
+We distinguish pinned installations based on exported (reproducible) environments and free installations where the installation is based on top-level dependencies listed in `requirements/requirements.yml`. If you start developing, you might want to do an unpinned installation and export the environment:
 
 ```bash
 tools/setup_env.sh -u -e -n <package_env_name>
 ```
 
 *Hint*: If you are the package administrator, it is a good idea to understand what this script does, you can do everything manually with `conda` instructions.
 
-*Hint*: Use the flag `-m` to speed up the installation using mamba. Of course you will have to install mamba first (we recommend to install mamba into your base
-environment `conda install -c conda-forge mamba`. If you install mamba in another (maybe dedicated) environment, environments installed with mamba will be located
+*Hint*: Use the flag `-m` to speed up the installation using mamba. Of course you will have to install mamba first (we recommend to install mamba into your base environment `conda install -c conda-forge mamba`). If you install mamba in another (maybe dedicated) environment, environments installed with mamba will be located
```

Comment:
libmamba is the default server for conda versions > 23.10. You could consider removing mamba from the setup script to make it leaner in a future PR .

Reply:
Done.